File: 241022/좀 더 어려운 수학 점수/math-scores-are-more-difficult.cpp
#include <iostream>
using namespace std;

int main() {
    int a_math, a_eng, b_math, b_eng;
    cin >> a_math >> a_eng >> b_math >> b_eng;

    if (a_math > b_math || (a_math == b_math && a_eng > b_eng)) {
        cout << "A";
    } else {
        cout << "B";
    }
    return 0;
}

File: 241022/좀 더 어려운 수학 점수/math-scores-are-more-difficult.js
const fs = require("fs");
let input = fs.readFileSync(0).toString().trim().split("\n");

let scoreA = input[0].split(" ");
let scoreB = input[1].split(" ");

let a_math = Number(scoreA[0]), a_eng = Number(scoreA[1]);
let b_math = Number(scoreB[0]), b_eng = Number(scoreB[1]);

if (a_math > b_math || (a_math === b_math && a_eng > b_eng)) {
    console.log("A");
} else {
    console.log("B");
}